### PR TITLE
Resolves #14

### LINKED ISSUE #14: Add guest to Google Calendar using email address
Is there any way now to add guest to Google Calendar event's using this library?

### PR DESCRIPTION
**Pull request by luqmanrom**
Resolve #14

### CODE REVIEW

**Comment by freekmurze** [comment]
Hi, have you tested this? Can you confirm that this works?

**Comment by luqmanrom** [comment]
Yeah. Tested on my current application, works fine.

File: src/Event.php [hunk 1]
```diff
@@ -195,6 +200,11 @@ protected function setDateProperty(string $name, Carbon $date)
         }
     }
 
+    public function addAttendees(array $attendees)
+    {
+        $this->attendees[] = $attendees;
```
Comment (sebastiandedeyne) [inline src/Event.php]: This looks off. Shouldn't it be either:

```php
$this->attendees = array_merge($this->attendees, $attendees);
```

or

```php
$this->attendees = $attendees;
```

(I prefer the former)

As it is now, you'll end up with an array of arrays.

**Comment by luqmanrom** [comment]
I think it should be an array of arrays. Refer [here](https://developers.google.com/google-apps/calendar/create-events)

**Comment by sebastiandedeyne** [comment]
I assume a single attendee is stored as an associative array. In that case, the method signature should also be singular:

```php
public function addAttendee(array $attendee)
{
    $this->attendees[] = $attendee;
}
```

**Comment by luqmanrom** [comment]
@sebastiandedeyne I agree in this case

**Comment by sebastiandedeyne** [comment]
Gonna merge this in and do the last changes myself. Thanks for the contribution!

**Pull request merged**